### ISSUE TITLE
Publication file download halts after a few hundreds of MB

### ISSUE THREAD

**Issue by nicolasfranck** [ugent-library/biblio-backend#61]
If you upload a big file (e.g. 1G zip), and try to download it,
the download halts around 800MB and then stops with
a network failure.

The request controller from biblio-backend reports this error:

```
2021/11/03 11:17:42 http: superfluous response.WriteHeader call from github.com/gorilla/handlers.(*responseLogger).WriteHeader (handlers.go:61)
2021/11/03 11:17:42 net/http: abort Handler
```

If you try to download the same file from the librecat backend server,
it works. Maybe missing headers?

**Comment by nicolasfranck** [comment]
Test against backend nginx with same files: works. Something with chunked response in librecat? The browser seems to be able to work with it, but golang does not..

**Comment by nicolasfranck** [comment]
https://github.com/ugent-library/biblio-backend/pull/66

**Comment by nics** [comment]
resolved by #66
